src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1173,7 +1173,8 @@ Glyphicon for success status messages
 .alert-success:before {
     font-family: 'Glyphicons Halflings';
     display: inline-block;
-    padding-right: 0.5em;
+    padding-top: .2em;
```

Comment:
> Other than that, not sure how to do that kind of search across the codebase

IntelliJ's find in path feature is very useful haha. Anyway after performing the search, it seems that `bootboxWrapper.es6` is the _only_ place that adds `alert` classes to modals!